arithmetic_analysis/in_static_equilibrium.py

```diff
@@ -13,7 +13,7 @@ def polar_force(
     Resolves force along rectangular components.
     (force, angle) => (force_x, force_y)
     >>> polar_force(10, 45)
-    [7.071067811865477, 7.0710678118654755]
+    [7.0710678118654755, 7.071067811865475]

```

Comment:
Do we need to use https://docs.python.org/3/library/math.html#math.isclose to make this test pass on multiple cpu architectures?

Reply:
We could use `round` instead.